.evergreen/generate_evergreen_tasks.js

```diff
@@ -404,11 +404,27 @@ for (const
 
     BUILD_VARIANTS.push({ name, display_name, run_on, expansions, tasks: taskNames });
   };
+
+  const configureLatestNodeSmokeTest = os.match(/^rhel/);
+  if (configureLatestNodeSmokeTest) {
```

Comment:
AC wasn't clear (shame on me for creating the ticket) but I chose to configure the latest node smoke test to only run rhel8.  I'm open to reworking it to run on more of we want to.

Reply:
Given that it's a smoke test I think its fine, but Windows would be "interesting" (so would macos but I think less so). Can we just make a note on the ticket (AC or comment) that we only tested on linux so its visible if folks look back at what the "latest" covered.